libutils/json.c

```diff
@@ -1353,6 +1353,21 @@ void JsonArrayAppendElement(
     SeqAppend(array->container.children, element);
 }
 
+void JsonArrayExtend(JsonElement *a, JsonElement *b)
+{
+    assert(a != NULL);
+    assert(a->type == JSON_ELEMENT_TYPE_CONTAINER);
+    assert(a->container.type == JSON_CONTAINER_TYPE_ARRAY);
+    assert(b != NULL);
+    assert(b->type == JSON_ELEMENT_TYPE_CONTAINER);
+    assert(b->container.type == JSON_CONTAINER_TYPE_ARRAY);
+
+    for (size_t i = 0; i < JsonLength(b); i++)
+    {
+        JsonArrayAppendElement(a, JsonCopy(JsonAt(b, i)));
```

Comment:
It might be useful to have a function that actually moves the array `b` to the end of the array `a`. Ideally by just merging the internal sequences and without extra allocations. Please check if that would be possible.

Reply:
Not sure if this is what you meant? :smile: